lib/rspec/matchers/built_in/change.rb

```diff
@@ -371,6 +371,14 @@ def evaluate_value_proc
           end
         end
 
+        def inspect_message_with_receiver(message, receiver)
+          if receiver.class == Class
```

Comment:
In RSpec codebase we prefer `Class === receiver` over `receiver.class == Class`, since the `receiver`, which is given by users, might be an object that does not respond to `#class` like BasicObject.